builder.go

```diff
@@ -115,10 +117,19 @@ func (b Builder) Build(ctx context.Context, outputFile string) error {
 		// support dlv
 		cmd.Args = append(cmd.Args, "-gcflags", "all=-N -l")
 	} else {
-		cmd.Args = append(cmd.Args,
-			"-ldflags", "-w -s", // trim debug symbols
-			"-trimpath",
-		)
+		if b.BuildFlags != "" {
+			// override build flags from environment if given
+			flags, err := shlex.Split(b.BuildFlags)
```

Comment:
I had the same assumption but during testing I found that go’s command interface interpreted all strings being part of the same argument which caused a failure during execution. 

I chose shlex because I would like to have the environment variable to support flags with variable arguments, e.g. ”-ldflags -w -s” and couldnt find a better solution, but I am open for other ideas.

Reply:
It could be that it is encoding problem and that we could drop shlex altogether but I am not too familiar how go handles strings, internally.

Reply:
Ah, right, I see what you mean...

Reply:
No worries, we can keep it